index.html

```diff
@@ -15,43 +15,75 @@
 </head>
```

Comment:
Don't forget to add your viewport meta tag in the head. 

https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag